### ISSUE TITLE
Which areas to work on?

### ISSUE THREAD

**Issue by AllThingsSmitty** [FrontendMasters/front-end-handbook#78]
@codylindley is there anything specific that you're looking to expand with this project?  Anything you want contributors to look into or focus on?

**Comment by codylindley** [comment]
Not yet, the book will get an update end of the year, for the next year. I'll get you involved then. 

Thanks, cody

However, I am wondering if how you'd feel about editing a React book? Any interest?

**Comment by AllThingsSmitty** [comment]
Yes, I'm interested. However, I only know React at a basic level. Do you mean help edit in the same capacity that I've done with this project, or are you looking for a more technical resource with deep React knoweldege?

**Comment by codylindley** [comment]
I'm looking for general editing. You game?

**Comment by AllThingsSmitty** [comment]
Yup, let's do this.

**Comment by SherylHohman** [comment]
I would also pitch in for some general editing..

**Comment by codylindley** [comment]
Howdy guys. The book is now public read it here: http://www.reactenlightenment.com/

Edit here: https://github.com/FrontendMasters/react-enlightenment

Any ?'s

**Comment by AllThingsSmitty** [comment]
None, other than do you want me to be checking for the same patterns that were adopted in the Front-End Handbook repo?

**Comment by codylindley** [comment]
Sure. And, a warning. The book is still very rough draft'ish in terms of grammar. I'm open to any changes or input here. Should mention also, the book is a work in progress I'm still drafting chapters.

**Comment by AllThingsSmitty** [comment]
Sounds good to me. 👍🏼

**Comment by codylindley** [comment]
Oops, wait. Give me 20mins more to fix urls.
